Contributing.md

```diff
@@ -18,12 +18,14 @@ Here's what you need to do to add or edit tutorials 👇:
    - Install the Python requirements with `pip install -r requirements.txt`
    - Install the pre-commit hooks with `pre-commit install`. This utility will run some formatting/checking
    tasks right before all git commit operations.
-2. If you're creating a new tutorial:
-   - Follow the [naming convention](#naming-convention-for-file-names) for file names.
-   - Add your new tutorial to [index.toml](/index.toml). Here, `weight` is the order in which your tutorial appears. For example, a tutorial with `weight = 15` comes after a tutorial with `weight = 10` and before `20`. Each tutorial comes with a Google Colab link and `Open in Colab` button on the top of the tutorial by default. If your new tutorial cannot be run on Google Colab, set `colab = false` not to display `Open in Colab` button on top the tutorial.
-3. Edit an existing tutorial or copy the [tutorial template](/tutorials/template.ipynb) to create a new tutorial. 
-4. Pre-commit hooks will ensure the `markdowns` folder reflects your changes but you can update the docs at any time:
-    - Run `python scripts/generate_markdowns.py --index index.toml --notebooks tutorials/your-tutorial.ipynb --output markdowns/`. This generates or updates the relevant markdown file in `/markdowns`.
+2. If you're **creating** a new tutorial:
+   - Create a copy of [tutorial template](/tutorials/template.ipynb) in `/tutorials` folder.
+   - Rename the new `.ipynb` file by following the [naming convention](#naming-convention-for-file-names).
+   - Follow the outline in the template as you create the tutorial.
+   - After the tutorial is complete, add necessary information to [index.toml](/index.toml). Here, `weight` is the order in which your tutorial appears. For example, a tutorial with `weight = 15` comes after a tutorial with `weight = 10` and before `20`. Each tutorial comes with a Google Colab link and `Open in Colab` button on the top of the tutorial by default. If your new tutorial cannot be run on Google Colab, set `colab = false` not to display `Open in Colab` button on top the tutorial.
+3. If you're **editing** an existing tutorial:
+   - Make necessary changes in the `.ipynb` file of the tutorial and save them. You don't need to make changes in the markdown file. 
+4. Pre-commit hooks will generate a corresponding `.md` file in `/markdowns` folder. Don't forget to add the new `.md` file to your `commit`. If you can't see any changes in `/markdowns`, run `python scripts/generate_markdowns.py --index index.toml --notebooks tutorials/your-tutorial.ipynb --output markdowns/` manually. This generates or updates the relevant markdown file.
```

Comment:
Pre-commit hooks will only check that the user has generated markdowns. It will not create and add it to the repo. So the previous like is a bit more accurate here.

Reply:
@TuanaCelik I updated the relevant section